d1985r0.md

```diff
@@ -394,6 +394,94 @@ Z<int> z1; // ok, decltype(z1)::value = int
 Z<1> z; // error, alias<1>::value is not a type
 ```
 
+## Overloading of class templates
+
+Universal template arguments enable what appears like overloading of
+class templates by implementing each overload as a specialization of a
+unimplemented primary template. As this is not real overloading there will be some
```

Comment:
"there will be some differences":

As this is specialization, and not overloading, CTAD is disabled, for instance. This is sometimes desirable - in other cases, deduction guides will have to be provided.